fc/parsing/actions.py

```diff
@@ -754,15 +745,13 @@ class OptionalVariable(BaseGroupAction, VariableReference):
     """
     def __init__(self, s, loc, tokens):
         super().__init__(s, loc, tokens)
-        if 'default' in self.tokens:
-            self.default_expr = s[self.tokens['default_start']:self.tokens['default_end']]
-        else:
-            self.default_expr = ''
+
+        self.default_expr = self.tokens.get('default', None)
```

Comment:
No `_expr` method needed?

Reply:
Seems to work fine! (Once I deleted all old caches)
Will add in if I get errors later